src/nih_wayfinding/app/scripts/profiles/profile-preference-options.js

```diff
@@ -39,13 +39,13 @@
 
             speedOpts: [{
                 text: 'About average (2.5 mph)',
-                value: 2.5
+                value: 1.1176
```

Comment:
Switching this will break the overview/navigation page calculations. Remind me to update them once this and my open PR are merged.